### ISSUE TITLE
When using $meteor Object fails on save if selector isn't the _id

### ISSUE THREAD

**Issue by vimes1984** [Urigo/angular-meteor#875]
when using: 
```
$meteor.object(Collection, {customfield: customvalue}, true);
```
when it's modified and saved angular-meteor-collection.js line 127: 
```
      this._updateParallel(selector, updates, callback);
``` 
gets passed the customvalue NOT the _id which casues mongo to fail with: 
```
Error: Not permitted. Untrusted code may only update documents by ID. [403]
    at throwIfSelectorIsNotId (http://localhost:3000/packages/mongo.js?69942a86515ec397dfd8cbb0a151a0eefdd9560d:480:11)
    at Mongo.Collection.(anonymous function) [as update] (http://localhost:3000/packages/mongo.js?69942a86515ec397dfd8cbb0a151a0eefdd9560d:660:9)
    at Mongo.Collection.(anonymous function) [as update] (http://localhost:3000/packages/aldeed_collection2.js?0e1a8fd75edeeb40b3ae5b365c9e75d0dc45d9a0:230:23)
    at http://localhost:3000/packages/angular-meteor-data.js?3aefb331daf81ba7d80cf798e204efe5242a9bce:730:27
    at Array.forEach (native)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?46eaedbdeb6e71c82af1b16f51c7da4127d6f285:149:11)
    at SubObject.AngularMeteorCollection._updateParallel (http://localhost:3000/packages/angular-meteor-data.js?3aefb331daf81ba7d80cf798e204efe5242a9bce:729:9)
    at SubObject.AngularMeteorCollection._updateDiff (http://localhost:3000/packages/angular-meteor-data.js?3aefb331daf81ba7d80cf798e204efe5242a9bce:716:12)
    at SubObject.AngularMeteorObject._updateDiff (http://localhost:3000/packages/angular-meteor-data.js?3aefb331daf81ba7d80cf798e204efe5242a9bce:1113:43)
    at SubObject.AngularMeteorObject.save (http://localhost:3000/packages/angular-meteor-data.js?3aefb331daf81ba7d80cf798e204efe5242a9bce:1095:14)(anonymous function) @ angular.js:12477(anonymous function) @ angular.js:9246Scope.$digest @ angular.js:15844Scope.$apply @ angular.js:16097(anonymous function) @ angular.js:23554jQuery.event.dispatch @ jquery.js:4665elemData.handle @ jquery.js:4333
````
Chris

**Comment by vimes1984** [comment]
the work around I found was to first find it then use th id of the found collection to use as the updater: 
```
    $scope.findroom = $meteor.object(Rooms, {stationID: pickedstattion._id}, false);
    $scope.room     = $meteor.object(Rooms, $scope.findroom._id, true);
```

**Comment by Urigo** [comment]
So, it's fixed in 1.3 but it's also not the best practice anymore, so you should start migrating to the `helpers` API with `findOne`.
Check out the new changes here:
http://info.meteor.com/blog/angular-meteor-1.3
